src/gui/transferlistwidget.cpp

```diff
@@ -1142,11 +1150,14 @@ void TransferListWidget::displayListMenu()
     if (addedPreviewAction)
         listMenu->addSeparator();
     if (oneHasMetadata)
-    {
         listMenu->addAction(actionForceRecheck);
-        listMenu->addAction(actionForceReannounce);
-        listMenu->addSeparator();
-    }
+    // We can not force reannounce torrents that are paused/errored/checking/missing files/queued.
+    // We may already have the tracker list from magnet url. So we can force reannounce torrents without metadata anyway.
+    listMenu->addAction(actionForceReannounce);
+    actionForceReannounce->setEnabled(oneCanForceReannounce);
+    if (!oneCanForceReannounce)
+        actionForceReannounce->setToolTip(tr("Can not force reannounce if torrent is Paused/Queued/Errored or in Checking state."));
```

Comment:
Having a period mark in tooltip looks strange for me. I would remove it.

Reply:
Maybe use this for brevity? 
```suggestion
        actionForceReannounce->setToolTip(tr("Can not force reannounce if torrent is Paused/Queued/Errored/Checking"));
```